torch/distributed/elastic/rendezvous/registry.py

```diff
@@ -14,8 +15,17 @@ def _create_etcd_handler(params: RendezvousParameters) -> RendezvousHandler:
     return etcd_rendezvous.create_rdzv_handler(params)
 
 
+def _create_expr_c10d_handler(params: RendezvousParameters) -> RendezvousHandler:
+    from .c10d_rendezvous_backend import create_backend
+
+    backend = create_backend(params)
+
+    return create_handler(backend.store, backend, params)
+
+
 def _register_default_handlers() -> None:
     handler_registry.register("etcd", _create_etcd_handler)
+    handler_registry.register("c10d-experimental", _create_expr_c10d_handler)
```

Comment:
What is the reason to name this `c10d-experimental` instead of just `c10d`? We usually tag things as "experimental" in the docs instead.

Reply:
Well I thought we should clearly indicate to the user that this is still a fairly new feature that might not be stable yet. If there is already a convention to convey such information solely in the docs, I can clearly change it to `c10d`. Will do that in the next revision.